governance.md

```diff
@@ -187,10 +187,12 @@ range of companies, universities and non-profit organizations. Because of this,
 it is possible that Members will have conflict of interests. Such conflict of
 interests include, but are not limited to:
 
--   Financial interests, such as investments, employment or contracting work,
+-   Financial interests, such as investments, employment, or contracting work,
     outside of The Project that may influence their work on The Project.
 -   Access to proprietary information of their employer that could potentially
     leak into their work with the Project.
+-   An issue where the person privately gains an advantage, but The Project
```

Comment:
I am +1, but this needs a bit more context.

For example, if a core developers wins the lottery they have personal gain, but the project has no gain (unless they donate). There is no conflict of interest in that case. And the project could even suffer in that case (let's imagine that the person decided to stop working as a result of the lotto $ - one less developer). The extra context needed is this - when the personal gain happens in a situation where official project resources are used. In those situations, this guideline is very helpful:

* A person decides to use project funds to take their family on a vacation to Italy - personal gain, no project gain = conflict of interest in the decision - they should recuse themselves from the decision.
* A person decides to attend a Jupyter team meeting with project funds - personal gain, but project gain as well - that person could be involved in the decision without conflict of interest.

Not sure what language to add to clarify this context.